stories/Overlays/Tooltip.stories.jsx

```diff
@@ -1,9 +1,11 @@
 import React, { forwardRef, useState } from "react";
 
 import { Button, Tooltip, Typography, Input } from "components";
+
+// eslint-disable-next-line import/extensions
```

Comment:
Instead of adding this to every file, add it to `overrides` in eslint configuration of this project.